packages/argo-checkout/src/extension-points/api/post-purchase/post-purchase.ts

```diff
@@ -49,6 +44,21 @@ interface Storage {
   update(data: any): Promise<void>;
 }
 
+interface InputData {
+  /** Identifier for the extension point */
+  extensionPoint: string;
+  /** Initial purchase */
+  initialPurchase: Purchase;
+  /** Checkout customer locale */
+  locale: string;
+  /** Shop where the checkout/order is from */
+  shop: Shop;
+  /** JWT representing the input_data payload */
+  token: string;
```

Comment:
For my understanding: `token` holds the entire signed `inputData` sans the token itself? 🤔

Reply:
That's correct, I could word the comments better, if that was the concern?

Reply:
No, the comments are fine, I just found it a bit surprising to have the token be part of the object that the token itself represents without itself being included.

I personally would have expected the token to be a sibling of inputData, but that's not a blocker of course.
Just wanted to be sure I understood correctly.

Reply:
It's how it had been implemented already with `initial_purchase`, with this change the token moves but is still contained inside the signed object. I can see the oddity of it for sure.